src/main/java/uk/gov/hmcts/reform/ccd/data/am/RoleAssignmentsQueryPostRequest.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.ccd.data.am;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@Getter
+@Setter
+public class RoleAssignmentsQueryPostRequest {
```

Comment:
We can rename and reuse RoleAssignmentsDeletePostRequest instead of creating a coppy.